flight/PiOS/inc/pios_modules.h

```diff
@@ -61,6 +61,13 @@ enum actuator_interlock {
 
 extern volatile enum actuator_interlock actuator_interlock;
 
+/* Ideally would have a better home for this constant */
+#define DSHOT_CMD_BEACON3 (3)
+
+extern int actuator_send_dshot_command(uint8_t cmd_id, uint8_t num_to_send,
+                uint16_t channel_mask);
+
+
```

Comment:
Ewww, none of this actuator stuff belongs here at all. :cry:

Reply:
We have no header, and worse-- no namespace-- for cross-module calls.  Thankfully we have very few.

A BeepDesired UAVO seems like overkill.

Reply:
That's because they're supposed to be independent and only communicate through UAVOs (I guess you can think of them as IPC), or system calls in PiOS.

Reply:
Yes, I agree in general, though there are other queues and paths and indirected ways of communications.

The other actuator one should not be via uavo (the actuator-inhibit).  This one is debatable-- I can reframe it that way if you want, though it'll consume resources.

Reply:
So is this PR doomed?

Reply:
No. If we can't work around it nicely I guess it has to be that way. :/